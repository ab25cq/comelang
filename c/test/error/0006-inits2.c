/*
PATTERN: end of struct
PATTERN: c:6:
*/

struct { int a; int b; int c; } x = {1, 2, 3, 4};

