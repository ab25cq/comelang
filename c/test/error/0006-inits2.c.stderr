end of struct already reached at test/error/0006-inits2.c:6:47
struct { int a; int b; int c; } x = {1, 2, 3, 4};
                                              ^
